jobbergate-core/tests/auth/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pendulum
+import pytest
+from jose.jwt import encode
+
+
+@pytest.fixture(scope="session")
+def time_now():
```

Comment:
You may be interested in the plummet library for testing with pendulum: https://github.com/dusktreader/plummet. The `frozen_time()` context manager allows you to freeze time at any given date.

Reply:
The only point I will make is that it's good to be able to see the timestamp where time is being frozen in the test code. This is especially true if you are comparing a `now()` value to some timestamp in test data. Otherwise, you're asking editors and reviewers to look up the project-wide freeze time in another test source file.